api/api-doc/task_manager.json

```diff
@@ -319,6 +319,18 @@
             "sequence_number":{
                "type":"long",
                "description":"The running sequence number of the task"
+            },
+            "shard":{
+               "type":"long",
+               "description":"The number of a shard the task is running on"
```

Comment:
This description is ambiguous, it sounds like `shard` is not a shard id, but a shard count: the count of shards this task is running on.

I would suggest: "The shard the task is running on".